packages/lockdown/package.json

```diff
@@ -12,9 +12,6 @@
     "lint-fix": "eslint --fix '**/*.js'",
     "lint": "eslint '**/*.js'"
   },
-  "devDependencies": {
-    "ava": "^5.1.1"
-  },
```

Comment:
Just a drive by, but I noticed this was completely unused. Are there any tools to detect dead dependencies? It would seem to be a simple static check.

Reply:
We use `depcheck` and if I understand it correctly, it should have caught this in its CI job. https://github.com/depcheck/depcheck